common/src/dodeca.rs

```diff
@@ -339,4 +383,29 @@ mod tests {
             epsilon = 1e-10
         );
     }
+
+    #[test]
+    fn perpendicular_vertex_is_complete() {
+        let mut error_count = 0_i32;
+
+        for s in 0..SIDE_COUNT {
+            let side = Side::from_index(s);
+            let incident_vertices = side.vertices();
+
+            #[allow(clippy::needless_range_loop)]
+            for v in 0..5 {
+                let vertex = incident_vertices[v];
```

Comment:
It's possible to get rid of `#[allow(clippy::needless_range_loop)]` here by using `for vertex in incident_vertices` instead of `for v in 0..5`.